src/dcqc/tests/base_test.py

```diff
@@ -195,12 +196,24 @@ def _interpret_process_outputs(self, outputs: dict[str, Path]) -> TestStatus:
         """Interpret the process output files to yield a test status."""
         exit_code = outputs["exit_code"].read_text()
         exit_code = exit_code.strip()
-        if exit_code == "0":
+        if exit_code == self.pass_code:
             status = TestStatus.PASS
         else:
             status = TestStatus.FAIL
         return status
 
+    # TODO: make changes to this package or the nextflow
+    # workflow so that file mounting is handled cleaner
+    @staticmethod
+    def _short_string_path(path: Path, substring: str) -> str:
+        # get index where staged folder is
+        index = [i for i, item in enumerate(path.parts) if substring in item][0]
```

Comment:
Nit: I don't foresee this happening, but what happens if the substring doesn't exist? (Since we are the ones always refining the substring)

Reply:
if the substring is not present in the path, it will error out on line 212 because the `index` will not exist.

Reply:
I updated the logic to make it a little cleaner (stopped using the 0 index logic) and to handle the case where the substring is not present in the path. Also added a test for that error raising in that case.